plugins/ProcessBase.h

```diff
@@ -45,6 +45,9 @@ class ProcessBase : public Host {
 
 	procptr_t peekPtr(const procptr_t address) const;
 
+	/// Resolves x64's RIP relative addressing.
+	procptr_t peekRIP(const procptr_t address) const { return address + peek< uint32_t >(address) + 4; }
```

Comment:
What does "RIP" stand for? I assume it doesn't mean "rest in peace"? :sweat_smile: 

Maybe just add the definition of RIP in parenthesis into the comment?